### PR TITLE
rcar_gen3: E3 target: fix compilation issues

### PR DESCRIPTION
**Pull request by ldts**
Target builds but has not been tested.

Signed-off-by: ldts <jorge.ramirez.ortiz@gmail.com>

### CODE REVIEW

File: plat/renesas/rcar/bl2_cpg_init.c [hunk 1]
```diff
@@ -42,7 +42,8 @@ static void bl2_secure_cpg_init(void)
 	uint32_t stop_cr2, reset_cr2;
 
 #if (RCAR_LSI == RCAR_E3)
-	reset_cr2 = 0x10000000U stop_cr2 = 0xEFFFFFFFU;
+	reset_cr2 = 0x10000000U;
+	 stop_cr2 = 0xEFFFFFFFU;
```
Comment (antonio-nino-diaz-arm) [inline plat/renesas/rcar/bl2_cpg_init.c]: nit: remove extra space

**Comment by antonio-nino-diaz-arm** [comment]
IP Review +1

File: plat/renesas/rcar/bl2_cpg_init.c (hunk 1 above)
Comment (ldts) [reply plat/renesas/rcar/bl2_cpg_init.c]: ok

**Comment by antonio-nino-diaz-arm** [comment]
jenkins: test this please

File: drivers/staging/renesas/rcar/ddr/ddr_a/ddr_init_e3.c [hunk 2]
```diff
@@ -1046,7 +1046,7 @@ uint32_t recovery_from_backup_mode(void)
    } else {
       NOTICE("[COLD_BOOT]");
    } /*  ddrBackup */
-   err=dram_update_boot_status(ddrBackup);
+   err=rcar_dram_update_boot_status(ddrBackup);
```
Comment (soby-mathew) [inline drivers/staging/renesas/rcar/ddr/ddr_a/ddr_init_e3.c]: nit: space needed around `=`.

Comment (antonio-nino-diaz-arm) [reply drivers/staging/renesas/rcar/ddr/ddr_a/ddr_init_e3.c]: This file doesn't follow at all the kernel coding style, there is no point in fixing specific problems here. It doesn't even use tabs for indentation.

**Pull request merged**